ocaml/xapi/xapi.ml

```diff
@@ -609,6 +611,32 @@ let check_network_reset () =
           else
             ""
         in
+        let mode_v6 =
+          match List.assoc_opt "MODE_V6" args with
+          | Some "static" ->
+              Some `Static
+          | Some "autoconf" ->
+              Some `Autoconf
+          | Some "dhcp" | Some _ ->
+              Some `DHCP
+          | None ->
+              None
+        in
+        let iPv6 =
+          if List.mem_assoc "IPV6" args then List.assoc "IPV6" args else ""
+        in
```

Comment:
This idiom avoid going through the list twice: 
```suggestion
        let iPv6 = Option.value ~default:"" (List.assoc_opt "IPV6" args) in
```

It could be used for the netmask and gateway as well.

Reply:
Yes, I can do it for the IPv4 fields as well, I was looking for such a pattern.